src/pages/docs/tutorials/tutorial-kitten-browser/app-setup/index.md

```diff
@@ -104,26 +104,29 @@ const AppBase = kind({
 			<div className={props.className}>
 				<div>
 					{/* For the feline-declined, you can replace the keyword below */}
-					<img src="//loremflickr.com/300/300/kitten" />
+					<img src="//loremflickr.com/300/300/kitten" alt="Kitten" />
 					<div>Kitten</div>
 				</div>
 			</div>
 		);
 	}
 });
 
-const App = MoonstoneDecorator(AppBase);
+const App = ThemeDecorator(AppBase);
 
 export default App;
-export {App, AppBase};
+export {
+	App, 
+	AppBase
+};
 ```
 
 ### JSX Hints
 
 You might be wondering why the `<img />` tag uses the self-closing syntax (`/>`). This is a
 [requirement imposed by JSX](https://facebook.github.io/react/tips/self-closing-tag.html), so
 whenever you include an element that doesn't have any children, you must either use the self-closing
-syntax or explicitly close it (`<img src=""></img>`).
+syntax or explicitly close it (`<img src=""></img>`). Also when you use the `<img />` tag, it is recommended to define an alt property for accessibility and failure to load the image. More info is available on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img).
```

Comment:
```suggestion
syntax or explicitly close it (`<img src=""></img>`). Also, when you use the `<img />` tag, it is recommended to define an alt property for accessibility failure to load the image. Detailed information is available on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img).
```

Reply:
I think this one should move to somewhere else.

Reply:
@YunhoLee0 Any ideas?

Reply:
I'm going to create "ESLint Hints" part including this one. Newly created part will be after "JSX Hints" part.

Reply:
Fixed in latest commit.